document/core/exec/instructions.rst

```diff
@@ -3077,66 +3077,69 @@ on the top of the stack.
 
  5. While the stack is not empty and the top of the stack is not an :ref:`exception handler <syntax-handler>`, do:
 
-    a. Pop the top element from the stack, prepending it to the :ref:`throw context <syntax-ctxt-throw>` of the exception: :math:`\XT[\val^n~(\THROWadm~a)]`.
+    a. Pop the top element from the stack.
 
  6. Assert: The stack is now either empty, or there is an exception handler on the top of the stack.
 
  7. If the stack is empty, then:
 
-    a. **TODO** *Return a result value representing the uncaught exception (will probably just be the same as 11.a.i. below).*
+    a. Return the uncaught exception :math:`\val^n~(\THROWadm~a)` as a value.
 
-.. todo::
-   After PR #221 is resolved, this step should be filled in with a PR to specify uncaught exception results.
+8. Else:
 
-8. Else there is an :ref:`exception handler <syntax-handler>` :math:`H` on the top of the stack.
+   a. Assert: there is an :ref:`exception handler <syntax-handler>` :math:`H` on the top of the stack.
```

Comment:
It seems off that this is under `else` but what follows is not.

Reply:
I am changing this to 

```
8. Else assert: there is an :ref:`exception handler <syntax-handler>` :math:`H` on the top of the stack.
```

This way I hope it's clear that everything that follows is under the else, given that the other branch of the "If" (`7.a`) returns. 

Or would you prefer that everything that follows is nested under `8.`?